.travis.yml

```diff
@@ -1,27 +1,100 @@
 language: rust
+sudo: false
 branches:
   only:
     - master
-os:
-  - linux
-  - windows
-rust:
-  - stable
-  - beta
-  - nightly
+
 # Need to cache the whole `.cargo` directory to keep .crates.toml for
 # cargo-update to work, but don't cache the cargo registry
 before_cache:
   - rm -rf /home/travis/.cargo/registry
 cache:
   directories:
     - /home/travis/.cargo
-before_script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then rustup component add clippy-preview; fi
-script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then cargo clippy --all-targets --all-features -- -D warnings; fi
-  - cargo test --verbose
+
+GH_TOKEN: &GH_TOKEN
+  secure:
+
+DEPLOY_TO_GITHUB: &DEPLOY_TO_GITHUB
+  before_deploy:
+    - git config --local user.name "Stuart Crust"
+    - git config --local user.email "stuart.crust@gmail.com"
+    - name="supernova-$TRAVIS_TAG-$TARGET"
+    - mkdir $name
+    - cp target/$TARGET/release/supernova $name/
```

Comment:
This should copy the actual build artifact into the folder `supernova-$TRAVIS_TAG-$TARGET`